script/Ciel/Day 3/03_01_CIEL03_3_3B.txt

```diff
@@ -674,12 +680,12 @@ No matter how unforgivable my sin...
 [sha:a6cca653bca725b10cd3e8b05f9ef3cfac1886dc]{
 -- Page 238, Offset 5461.
 -- 俺は、その言葉に救われた。
-I'm saved by her words.
+Her words prove my salvation.
```

Comment:
Feels like it's trying too hard and makes it kinda unclear in EN without the jp to read.